dashboard/app/controllers/shared_blockly_functions_controller.rb

```diff
@@ -4,6 +4,7 @@ class SharedBlocklyFunctionsController < ApplicationController
   load_and_authorize_resource
 
   def new
+    @shared_blockly_function.block_type = 'behavior'
```

Comment:
I think this is covered by the `before_save` in the model?

https://github.com/code-dot-org/code-dot-org/blob/1c67af3e880d9568c7fb52a68714012091bfe8e1/dashboard/app/models/shared_blockly_function.rb#L24-L27

Reply:
Ah I see — removed below.